ansible_base/resource_registry/views.py

```diff
@@ -141,6 +141,10 @@ def manifest(self, request, name, *args, **kwargs):
             return HttpResponseNotFound()
 
         resources = Resource.objects.filter(content_type__resource_type=resource_type).prefetch_related("content_object")
+
+        if name == "shared.user":  # Skip the _system user
+            resources = resources.exclude(name="_system")
```

Comment:
Prefer `settings.SYSTEM_USERNAME` instead of hardcoding `_system`

Reply:
Interesting, galaxy_ng even when running with DAB profile does not have that `settings.SYSTEM_USERNAME` so I guess I must use `getattr(settings, "SYSTEM_USERNAME", "_system")`

Reply:
Makes sense since that is not in [dynamic settings](https://github.com/ansible/django-ansible-base/blob/devel/ansible_base/lib/dynamic_config/dynamic_settings.py), and I don't think galaxy_ng _should_ define that setting, because like AWX, it is not using this construct of a system user.

Alternative to the `getattr` - it may be better to not exclude this user from the list if the setting is not set.